source/direct_debit/index.html.md.erb

```diff
@@ -176,13 +176,13 @@ For example:
 
 The query parameters you can use to search are:
 
-- `reference`
-- `status`
-- `bank_statement_reference`
+- `reference` - exact match only
+- `status` - exact match only
+- `bank_statement_reference` - exact match only
 - `email`
 - `name`
 
-All the parameters are case insensitive. You can search for partial values of `reference`, `email` and `name`.
+All the parameters are case insensitive.
```

Comment:
I think `state` is case sensitive.

Reply:
Thank you - fixed in https://github.com/alphagov/pay-tech-docs/pull/320/commits/ec5793bfcb5ccd6af48b5f7977c3e98505ec0253.